packages/manager/tools/component-rollup-config/src/plugins/translation-ui-router.ts

```diff
@@ -92,7 +92,7 @@ export = (opts:any = {}) => {
                 format,
               );
 
-              inject = `translations: ($q, $translate, asyncLoader) => { ${inject} }`;
+              inject = `translations: /* @ngInject */ ($q, $translate, asyncLoader) => { ${inject} }`;
```

Comment:
Updated on develop but target branch feat/web-ca needs to be rebased. Only purpose of updating it in this PR is to be able to check that branch builds and that behaviour is correct on a prod environment